traits/testing/tests/test_unittest_tools.py

```diff
@@ -17,7 +17,7 @@
 
 from traits import _py2to3
 
-from traits.testing.unittest_tools import unittest
+import unittest
```

Comment:
This one should actually stay as-is, to check that we don't break downstream packages doing "from traits.testing.unittest_tools import unittest".

Reply:
Fixed.